test/distributed/test_symmetric_memory.py

```diff
@@ -287,6 +287,41 @@ def test_wait_signal_timeout(self) -> None:
         # impossible to terminate the process in this state.
         os._exit(0)
 
+    @skipIfRocm
+    @requires_cuda
+    def test_allow_overlapping_devices(self) -> None:
+        os.environ["TORCH_SYMM_MEM_ALLOW_OVERLAPPING_DEVICES"] = "1"
```

Comment:
How do distributed tests work in PyTorch? Do we need to clear this env var at the end of the test case to avoid it "polluting" other test cases in case the same process gets reused?

Reply:
Thanks for checking! This test class is a `MultiProcessTestCase` which runs test methods in subprocesses.